### ISSUE TITLE
Command line options fed to ssh

### ISSUE THREAD

**Issue by ghost** [claranet/ssha#16]
Options listed here https://github.com/claranet/ssha/blob/master/ssha/ssh.py#L20 may be problematic for connection security and stability.

The `-A` option ( authentication agent forwarding ) is something that is considered harmful. Should the bastion/jumphost be compromised it would allow someone on that machine to make use of the keys contained in your local ssh key agent.

The `-t` option ( Force pseudo-terminal allocation ) may cause the connection to become unstable or drop during the transfer of binary content. 

Alternate ways of doing this would be to either:

Use the `-J` option - a shortcut for the `ProxyJump` option ( openssh ver. 7.3+ Required ):

```shell
ssh -J <jumphost> <target>

```

The ProxyCommand option using netcat on the jump host:

```shell
ssh -o ProxyCommand="ssh %h nc <target> 22" <jumphost>
```

or the ProxyCommand with the `-W` option:

```shell
ssh -o ProxyCommand="ssh -W %h:%p <jumphost>" <target>

```

The proxycommand or ProxyJump configuration directives could also be placed in the ssh configuration file for the same effect

**Comment by raymondbutcher** [comment]
Yeah, the `-A -t` was a quick way to get it working so I could focus on other features.

I've now done some of the ground work that will help with this. So I will add a sensible default to ssha and allow changing it in 2 places:

* the `.ssha` file that is specific to a project
* the `~/.config/ssha/config` file that a user can use to override settings globally or per-project
  * not yet implemented, see #9

My thoughts on the "sensible" default are:

* the `-J` option is too new (I don't have it) so we can't rely on that.
* I remember once trying to use `nc` and finding that it wasn't installed on the bastion host
* I don't remember ever having a problem with `ProxyCommand/-W` so this seems like the way to go.

Does that sound good?

Thanks for laying out the options, it was very helpful!

**Comment by raymondbutcher** [comment]
Done in #20
